Formula/kumo.rb

```diff
@@ -0,0 +1,24 @@
+class Kumo < Formula
+  desc "Kumo: Word Clouds in Java"
+  homepage "https://github.com/kennycason/kumo"
+  url "https://search.maven.org/remotecontent?filepath=com/kennycason/kumo-cli/1.12/kumo-cli-1.12.jar"
+  sha256 "9d67d943676be57fe75d47f1832c4e8cb415ee8e4e82a71d4d7d1e7b4effe7c4"
+
+  depends_on :java => "1.8+"
+
+  def install
+    libexec.install "kumo-cli-1.12.jar"
+    bin.write_jar_script libexec/"kumo-cli-1.12.jar", "kumo"
+    puts "Finished installing kumo 1.12"
+  end
+
+  def server_script(server_jar); <<-EOS.undent
+    #!/bin/bash
+    exec java -cp #{server_jar} com.kennycason.kumo.cli.KumoCli "$@"
+  EOS
+  end
+
+  test do
+    pipe_output("#{bin}/kumo --version", "Test Kumo version command")
```

Comment:
We need a test that does more than that… like writes an actual image

Reply:
Those tests are already handled in the code base. Those seem too specific to be part of the tool installation validation. It's easy enough to add, but I question the value. Asserting the jar is accessible and responding seems sufficient.

To test that from the CLI tool, I'd have to A) load from a fixed url which I'm ok with but I'd pick something like wikipedia.org, not sure if that's ok, or B) provide a static file, which is less doable, and C) pick some safe place to output the generated image (/tmp/?).

To make the test be any more valuable than the print version, I would also need to form some sort of assert to verify the image contents of the word cloud were "correct", which isn't really doable.

Reply:
If this is decided to be absolutely required, I have prepared:
```
pipe_output("#{bin}/kumo --version", "Test Kumo version command")
pipe_output("#{bin}/kumo -i https://wikipedia.org -o /tmp/wikipedia.png", "Generate simple wordcloud")
```